Formula/pike.rb

```diff
@@ -36,6 +42,10 @@ def install
     # Workaround for https://git.lysator.liu.se/pikelang/pike/-/issues/10058
     ENV.append "CFLAGS", "-Wno-implicit-function-declaration"
 
+    # Use GNU sed on macOS to avoid this build failure:
+    # sed: RE error: illegal byte sequence
+    ENV.prepend_path "PATH", Formula["gnu-sed"].libexec/"gnubin" if OS.mac?
```

Comment:
Can we report this upstream?

Reply:
I opened an issue on the Pike bug tracker and linked to it in the comment.